.readthedocs.yaml

```
@@ -0,0 +1,17 @@
+# https://docs.readthedocs.io/en/stable/config-file/v2.html#supported-settings
+
+version: 2
+
+sphinx:
+  builder: html
+
+build:
+  os: "ubuntu-20.04"
+  tools:
+    python: "3.9"
+    rust: "1.55"
+
+python:
+  install:
```

To make sure it installs the right doc requirements:

```suggestion
  install:
    - requirements: docs/requirements.txt
```

See also https://github.com/clbarnes/ncollpyde/pull/22